x-pack/plugins/security_solution/public/common/hooks/use_url_state.ts

```diff
@@ -30,5 +30,6 @@ export enum URL_PARAM_KEY {
   sourcerer = 'sourcerer',
   timeline = 'timeline',
   timerange = 'timerange',
+  pageFilter = 'pageFilters',
```

Comment:
I wonder if we should name this something more specific to the alerts page or keep this generic in case we want to use these filters on another page (i.e. events?).. Just thinking aloud

Reply:
@michaelolo24 Actually, this param key is tied to to the filter group component which can be reused across other pages as well. May be we can rename it as `filterGroup` or anything else. 

Additionaly, I can take param key as the prop the component so that implementer can choose it. what do you think?

Reply:
Nah, no reason to, better to remain consistent across the pages. Thanks!